introduction.md

```diff
@@ -0,0 +1 @@
+hello, is it _me_ you're looking for?
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.